openml/evaluations/functions.py

```diff
@@ -172,6 +172,12 @@ def __list_evaluations(api_call, output_format='object'):
         type(evals_dict['oml:evaluations'])
 
     evals = collections.OrderedDict()
+    uploader_ids = list(set([eval_['oml:uploader'] for eval_ in
+                             evals_dict['oml:evaluations']['oml:evaluation']]))
+    api_users = "user/list/user_id/" + ', '.join(uploader_ids)
```

Comment:
Is there a reason for the whitespace between the user IDs? Wouldn't it be better to not have these for http requests?